src/main.c

```diff
@@ -14,11 +14,12 @@
  */
 static void print_help(void)
 {
-    puts("Usage: " APP_NAME " [OPTION...] FILE...");
+    puts("Usage: " APP_NAME " [OPTION...] [FILE|DIR]...");
```

Comment:
The square brackets tell us that FILE|DIR is optional. In fact, this is not the case.

Reply:
You're right, this is not correct. But, thinking about it, we could make it correct. What if calling swayimg without any file or directory would open current directory? It would be very easy to implement and I think such behavior would feel quite natural (but should be mentioned in man page anyway).

Reply:
Implemented in 637e6b7895ec352bbeb9af8f561ce89767979d82.